### ISSUE TITLE
Starting chromedriver on port 9515 Only local connections are allowed.

### ISSUE THREAD

**Issue by elgalu** [elgalu/docker-selenium#120]
Seeing this error from time to time:

```sh
Starting chromedriver on port {{some_port}} Only local connections are allowed.
```

The hack `DBUS_SESSION_BUS_ADDRESS=/dev/null` seems to work but is not exactly correct, chromedriver log complains `/dev/null` is not a valid value for that.

**Comment by elgalu** [comment]
Found a way without having to share `/var/run/dbus`

```sh
# Dockerfile
sudo apt-get -qyy install dbus-x11
# entry.sh
sudo rm -f /var/lib/dbus/machine-id
sudo mkdir -p /var/run/dbus
sudo service dbus restart
service dbus status
export $(dbus-launch)
export NSS_USE_SHARED_DB=ENABLED
echo "-- INFO: DBUS_SESSION_BUS_ADDRESS=${DBUS_SESSION_BUS_ADDRESS}"
#=> e.g. DBUS_SESSION_BUS_ADDRESS=unix:abstract=/tmp/dbus-APZO4BE4TJ,guid=6e9c098d053d3038cb0756ae57ecc885
echo "-- INFO: DBUS_SESSION_BUS_PID=${DBUS_SESSION_BUS_PID}"
#=> e.g. DBUS_SESSION_BUS_PID=44
```

**Comment by elgalu** [comment]
Fix released on [2.53.1y](https://github.com/elgalu/docker-selenium/releases/tag/2.53.1y)
